server/master/src/main/java/org/apache/accumulo/master/TabletGroupWatcher.java

```diff
@@ -611,12 +609,11 @@ private void deleteTablets(MergeInfo info) throws AccumuloException {
       TabletsSection.ServerColumnFamily.TIME_COLUMN.fetch(scanner);
       scanner.fetchColumnFamily(DataFileColumnFamily.NAME);
       scanner.fetchColumnFamily(TabletsSection.CurrentLocationColumnFamily.NAME);
-      Set<FileRef> datafiles = new TreeSet<>();
+      Set<String> datafiles = new TreeSet<>();
       for (Entry<Key,Value> entry : scanner) {
         Key key = entry.getKey();
         if (key.compareColumnFamily(DataFileColumnFamily.NAME) == 0) {
-          datafiles
-              .add(new FileRef(TabletFileUtil.validate(key.getColumnQualifierData().toString())));
+          datafiles.add(key.getColumnQualifierData().toString());
```

Comment:
Why drop validation here?

Reply:
Hmm this might have been a mistake.  I had this code using TabletFile at one point but then had to revert to Strings because of the GC code, per my other comment.

Reply:
Fixed in 3aacfe603ad9c5d432df1da6411297783adba004